data/campaigns/Heir_To_The_Throne/scenarios/00_The_Great_Continent.cfg

```diff
@@ -0,0 +1,1523 @@
+        {PREPARE_MESSAGE ( {VARIABLE_CONDITIONAL status_s10  not_equals completed}     {VARIABLE_CONDITIONAL status_s11  not_equals completed}  ) _"Reglok, Elensefar thieves’ guildmaster."}
+        {PREPARE_MESSAGE ( {VARIABLE_CONDITIONAL status_s08b not_equals completed}                                                              ) _"Harper, Three Sisters outlaw."}
+
+        # if we have at least 2 options, say the whole message. If we have only 1 option, use a different, summarized message
+        {IF} {VARIABLE_CONDITIONAL message_count greater_than_equal_to 2} {THEN(
+            [message]
+                speaker=crier
+                #po: $message is a list of names and titles, generated depending on Konrad's actions. For example: "Maddock, Lord of Elensefar. Ethiliel, Lady of the Aethenwood. Delurin, Three Sisters outlaw. "
+                message=_"Hark! The Lord Konrad and his followers are to be made aware of the events to proceed at Weldyn in three weeks’ time, these being the just and righeous execution of the following condemned criminals and rebel sympathizers:
+
+$message|And one thousand twenty-eight other rebel partisans from the western and northern reaches."
+            [/message]
+        )} {ELSE(
+            [message]
+                speaker=crier
+                message=_"Hark! The Lord Konrad and his followers are to be made aware of the events to proceed at Weldyn in three weeks’ time, these being the just and righeous execution of one thousand twenty-eight condemned partisans and rebel sympathizers from the western and northern reaches."
+            [/message]
+        )} {/IF}
+        {CLEAR_VARIABLE message,message_count}
+        [message]
+            speaker=crier
+            message=_"The Queen laments such a loss of life as regrettable, but necessary in Her judgement to ensure the continued safety and stability of Her kingdom.
+
+The Lord Konrad is warned to discontinue his campaign of terror and keep well away from Weldyn, lest the same tragic fate befall him as well."
+        [/message]
+        {MOVE_UNIT id=crier 56 45}
+        {KILL id=crier}
+        {DELAY 750}
+
+        #------------------------
+        # DISCUSSING WHAT TO DO
+        #------------------------
+        [message]
+            speaker=Konrad {KONRAD_VARIATION concerned}
+            message=_"A mass execution!? Sweet Haldric, all those people. We have to act!"
+        [/message]
+        [message]
+            speaker=Konrad {KONRAD_VARIATION concerned}
+            message=_"...but what did Asheviere hope to gain by telling us? Could she mean to catch us in some trap?"
+        [/message]
+        [message]
+            speaker=Lisar {LISAR_VARIATION mad}
+            #po: Li'sar's "Gee, you think?" is sarcastic
+            message=_"Gee, you think? Of course it’s a trap!"
```

Comment:
Modern expression of 'you think' aside, even the 'gee' sounds strange from Li'sar who is supposed to have a noblewoman and princess' character, would you concur?